app/views/catalog/_ot_add.html.haml

```diff
@@ -33,9 +33,13 @@
         .col-md-8
           = select_tag("type",
                        options_for_select({"Amazon CloudFormation" => "OrchestrationTemplateCfn",
```

Comment:
would be good if we can ask the OrchestrationTemplate model to supply all the data for the description: class_name hash.